test-flip-link-app/Cargo.toml

```diff
@@ -9,7 +9,15 @@ version = "0.1.0"
 cortex-m = "0.7"
 cortex-m-rt = "0.6"
 cortex-m-semihosting = "0.3"
-lm3s6965 = "0.1"
 panic-semihosting = { version = "0.5", features = ["exit"] }
 
+# optional
+lm3s6965 = { version = "0.1", optional = true }
```

Comment:
I think (but double check) that marking a dependency as optional automatically creates a Cargo feature with the same name: so this line should create a "lm3s6965" feature which you can use in `#[cfg]` statements.

Reply:
Yes. This works!